src/api/implementation/configuration/versions.ts

```diff
@@ -1,10 +1,20 @@
+/* eslint-disable camelcase */
+
 import * as v1 from "./v1";
+import * as v1_1 from "./v1_1";
 import { API } from "../../contract/api";
 import { versionUnknown, available } from "../apiutils";
+import { EventEmitter } from "vscode";
+import { ActiveValueTracker } from "../../../components/contextmanager/active-value-tracker";
+import { ConfigurationV1 } from "../../contract/configuration/v1";
 
-export function apiVersion(version: string): API<any> {
+export function apiVersion(version: string,
+    onDidChangeKubeconfigEmitter: EventEmitter<ConfigurationV1.KubeconfigPath>,
```

Comment:
It works the same but i would suggest to use the KubeconfigPath from `kubectl/kubeconfig`

```
import { KubeconfigPath } from "../../../components/kubectl/kubeconfig"; 

onDidChangeKubeconfigEmitter: EventEmitter<KubeconfigPath>
```